.yarnrc

```diff
@@ -2,7 +2,7 @@
 # yarn lockfile v1
 
 
-registry "https://registry.npmjs.org/"
+registry "http://artifacts.netflix.com/api/npm/npm-netflix"
```

Comment:
This and "netflix" release versions are probably committed accidently? 😅

Reply:
Oops, I pushed my internal release commits.  Cleaned up with a force push.  Thanks!